### ISSUE TITLE
[QA][Sharing-NG] Add test coverage of getting a single permission

### ISSUE THREAD

**Issue by amrita-shrestha** [owncloud/ocis#9524]
**JIRA:** [#OCISDEV-21 ](https://kiteworks.atlassian.net/browse/OCISDEV-21)

Missing tests coverage of getting a single permission for root item/ resource.
- [x] https://owncloud.dev/libre-graph-api/#/drives.root/GetPermissionSpaceRoot
    - https://github.com/owncloud/ocis/pull/11337
- [x] https://owncloud.dev/libre-graph-api/#/drives.permissions/GetPermission
    - https://github.com/owncloud/ocis/pull/11345

**Comment by ghost** [comment]
~waiting for ~https://github.com/owncloud/ocis/issues/9541~ https://github.com/owncloud/ocis/issues/8616 to be implemented. blocked until then~
UPDATE: we are adding the basic tests for the endpoints

**Comment by deyankiteworks** [comment]
> waiting for ~[#9541](https://github.com/owncloud/ocis/issues/9541)~ [#8616](https://github.com/owncloud/ocis/issues/8616) to be implemented. blocked until then

I asked @2403905 about [#8616](https://github.com/owncloud/ocis/issues/8616) and he told me there is a work around in [#9541](https://github.com/owncloud/ocis/issues/9541).

> We can retrieve an item permissions via [ListPermissions API](https://owncloud.dev/libre-graph-api/#/drives.permissions/ListPermissions)

**Comment by saw-jan** [comment]
> > waiting for ~[#9541](https://github.com/owncloud/ocis/issues/9541)~ [#8616](https://github.com/owncloud/ocis/issues/8616) to be implemented. blocked until then
> 
> I asked [@2403905](https://github.com/2403905) about [#8616](https://github.com/owncloud/ocis/issues/8616) and he told me there is a work around in [#9541](https://github.com/owncloud/ocis/issues/9541).
> 
> > We can retrieve an item permissions via [ListPermissions API](https://owncloud.dev/libre-graph-api/#/drives.permissions/ListPermissions)

yeah, we can use the workaround to get the single permissions. This ticket is to add API tests for the single permission endpoints which are now added. Therefore, closing this here
